superset/connectors/base/models.py

```diff
@@ -533,6 +533,7 @@ class BaseColumn(AuditMixinNullable, ImportExportMixin):
     def __repr__(self) -> str:
         return str(self.column_name)
 
+    bool_types = ("BOOL",)
```

Comment:
add more general column type
```
    bool_types = ("BOOL", "BOOLEAN", )
    num_types = ("TINYINT", )
    date_types = ("TIMESTAMP", )
    str_types = ("TEXT", "MEDIUMTEXT", "LONGTEXT", )
```

Reply:
Actually these aren't really used anywhere, it's just a fallback non-SQLA connectors; the models in `sqla/models.py` file checks these directly from the relevant db engine spec, hence the relevant definitions are defined in `BaseEngineSpec` and can be overridden on an engine-by-engine basis.